Makefile

```diff
@@ -13,7 +13,7 @@ verify: clean
 
 docker: clean
 	mvn install package dockerfile:push -DskipTests=true -Dtag=${tag} \
-	  -DgcrRepo=${repo} --file ${name}-sample/pom.xml
+	  -DgcrRepo=${repo} --file secure-api-gateway-ob-uk-rcs-sample/pom.xml
 
```

Comment:
update the name variable instead? Or remove it

Reply:
The variable name was used to create the helm charts too, and then I have fix only the path to the pom file